src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -108,11 +142,18 @@ public bool IsMetadataVirtualLocked
                 get { return (_flags & IsMetadataVirtualLockedBit) != 0; }
             }
 
+            public bool IsVarArg
+            {
+                get { return (_flags & IsVarArgBit) != 0; }
+                set { ThreadSafeFlagOperations.Set(ref _flags, value ? IsVarArgBit : 0); }
+            }
```

Comment:
note: IsVarArg is currently lazily computed.  But it woudl be trivial to compute up front in SourceOrdinaryMethod's constructor and just not have to deal with this.  But i kept the logic as before.